securityscanutils/trivy_scanner_test.go

```diff
@@ -104,9 +104,21 @@ var _ = Describe("Trivy Scanner", func() {
 			inputImage = "quay.io/solo-io/gloo:1.11.1"
 			samples := 8
 			for i := 0; i < samples; i++ {
-				_, _, err := t.ScanImage(context.TODO(), inputImage, inputMarkdownTemplateFile, outputFile)
+				_, _, err := t.ScanImage(context.TODO(), inputImage, inputMarkdownTemplateFile, outputFile, "")
 				Expect(err).NotTo(HaveOccurred())
 			}
 		})
+
+		It("ignores CVEs in trivyIgnore", func() {
```

Comment:
I think this test should first run a scan, and produce vulnerabilities. Then we can run another scan with the ignore file and demonstrate that with the file, those same vulnerabilities are not discovered.

What happens if 1.9.21 finds new vulnerabilities 1 month from now? Won't this test start failing because we will now find vulnerabilities? How do we want to address that?